samples/net/wifi/overlay-enterprise.conf

```diff
@@ -0,0 +1,6 @@
+CONFIG_WIFI_NM_WPA_SUPPLICANT_CRYPTO_ENTERPRISE=y
+# For EAP frames
+CONFIG_NET_BUF_DATA_SIZE=1100
```

Comment:
This is not good to set the net_buf so large. The code should work with default value (128). We (the core network code), should not dictate the size of the net_buf data part as that could affect how the end user is able to use the networking APIs.

Reply:
Well, the EAP packets are each greater than 1K and if we use 128 buffer we need large number of packets and buffers to sustain the Enterprise exchange.

Reply:
The problem is with small packets, we waste lots of memory if using buf size over 1k (when fixed buffers are used).

Reply:
Yes, that is I have put this only in the Enterprise overlay, I can try and see the settings with which Enterprise handshake works reliably with default network stack setttings, but that would need a few trial and error experiments and would still be inefficient use of network buffers for the handshake.

Reply:
https://www.cloudshark.org/captures/5a5bc254b19d See the EAP frames they are ~1100 byte size.

> The problem is with small packets, we waste lots of memory if using buf size over 1k (when fixed buffers are used).

I think it's time to make variable buffer mainstream don't you think? Shall I enable that in this overlay?

Reply:
BTW, I tried again with default network buf settings, and see below:

```
[00:01:23.846,801] <err> net_pkt: Data buffer (1407) allocation failed.
[00:01:23.846,832] <wrn> net_conn: pkt cloning failed, pkt 0x200588c0 dropped
[00:01:24.682,312] <dbg> wpa_supp: wpa_printf_impl: EAPOL: startWhen --> 0
[00:01:26.804,382] <err> net_pkt: Data buffer (1407) allocation failed.
[00:01:26.804,412] <wrn> net_conn: pkt cloning failed, pkt 0x200588c0 dropped
```

And using variable buffer + 1100 works with default variable buffer pools, so, IMHO, we can go ahead with that. WDYT?

Reply:
ok, as discussed added both fixed and variable buffer overlays.